cdap-standalone/src/main/java/io/cdap/cdap/StandaloneMain.java

```diff
@@ -523,7 +524,8 @@ static StandaloneMain create(CConfiguration cConf, Configuration hConf) {
       new LocalLogAppenderModule(),
       new LogReaderRuntimeModules().getStandaloneModules(),
       new RouterModules().getStandaloneModules(),
-      new SecurityModules().getStandaloneModules(),
+      new CoreSecurityModules().getStandaloneModules(),
+      new ExternalAuthenticationModule(),
```

Comment:
Are there a use case where this guys don't go together? If no, we can add AuthenticationModule inside of  CoreSecurityModules

Reply:
I tried to add CoreSecurityModules originally as a set of dependencies in the ExternalAuthenticationModule and AuthenticationContextModules, but this caused binding conflicts in places in which both were used. We can try to refactor it later